src/Model/PowerElectronics/MicrogridLoad/MicrogridLoad.cpp

```diff
@@ -44,9 +44,9 @@ namespace GridKit
   template <class ScalarT, typename IdxT>
   int MicrogridLoad<ScalarT, IdxT>::allocate()
   {
-    y_.resize(size_);
-    yp_.resize(size_);
-    f_.resize(size_);
+    y_.resize(static_cast<size_t>(size_));
+    yp_.resize(static_cast<size_t>(size_));
+    f_.resize(static_cast<size_t>(size_));
```

Comment:
Let's make it a little less cluttered. Consider something like this here and in similar places throughout the code.
```suggestion
    auto size = static_cast<size_t>(size_);
    y_.resize(size);
    yp_.resize(size);
    f_.resize(size);
```